configs/template_config.yaml

```diff
@@ -43,16 +43,21 @@ model_path_dir: ~
 # Example: 4-1-retnet-7B
 model_label: ~
 
+# Checkpoint Path (str): Path to checkpoint to load for generation/inference/latency tests.
```

Comment:
I added a comment that it is used in latency tests and then moved this arg up into this section because it pertains much more to the `TESTING AND GENERATION CONFIGURATION` section of the yaml than the `DATASET AND PATH CONFIGURATION` section. I noticed that I had to keep flipping between the two secitons to check both this arg and `model_path_dir` and realized `checkpoint_path` didn't even relate to the latter section.